react.js

```diff
@@ -0,0 +1,68 @@
+/** @type {import("eslint").Linter.Config}  */
+const react = {
+  extends: [
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@tanstack/eslint-plugin-query/recommended',
```

Comment:
I think we need to install or remove this plugin

Reply:
I added missing libs into package.json :)